### ISSUE TITLE
NEWS file

### ISSUE THREAD

**Issue by pbouffard** [olivierlacan/keep-a-changelog#246]
Found this resource with a search something like "CHANGELOG skeleton in Markdown" and got exactly what I was looking for, thanks!

I think it would also be helpful to be the place where "NEWS file skeleton in Markdown" searches end up--the first page of results for that search doesn't yield anything quite as useful, and also I think a NEWS file (or whatever you want to call release notes) is a useful instrument especially to those who aren't interested in tracking finer-grained updates.

**Comment by morinim** [comment]
*NEWS.md* could also be a suggested file name.

The [GNU NEWS](https://www.gnu.org/prep/standards/standards.html#NEWS-File) file format is pretty close in spirit to what's being advocated.

I know that "The GNU style guide is a nice start but it is incredibly naive", but it seems like an opportunity for keepachangelog.com.

Perhaps it could be enough to change the the paragraph:

> What should the changelog file be named? Call it `CHANGELOG.md`

to support also `NEWS` / `NEWS.md`

PS thanks for the outstanding work!

**Comment by olivierlacan** [comment]
We've debated this issue many times before. GNU and Debian are not divine law. Their conception of a changelog is flawed and [I've explained in Keep a Changelog itself][1] that I don't think the `NEWS` naming is as clear and appropriate for what I (and many others in the open source community) think a changelog should offer to end users.

I really like [@while0pass's summary][2] of the differences between all layers of communication on a project. It reminds me of the [OSI model][3] a bit. I think there's a level of editorial newsworthiness required for changelog items to be included within release notes that is a bit or much higher. As @while0pass astutely pointed out, the intended audience for changelogs is often much more technical than the usual intended audience for release notes. Even if we're talking about developer tooling.

[1]: https://keepachangelog.com/en/1.0.0/#standard
[2]: https://github.com/olivierlacan/keep-a-changelog/issues/256#issuecomment-415556967
[3]: https://en.wikipedia.org/wiki/OSI_model
